azure_arc_k8s_jumpstart/cluster_api/capi_keyvault_extension/capi_keyvault_k8s_extension.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export keyVaultResourceGroup='<KeyVault Resource Group name>'
+export keyVaultLocation='<Key Vault Location>'
+export resourceGroup='<Azure Arc CLuster Resource Group name>'
+export arcClusterName='<Azure Arc Cluster Name>'
+export namespace='hello-arc'
+export k8sExtensionName='sscsi'
+export keyVaultName=secret-store-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
+
+# Installing Helm 3
+echo "Installing Helm 3"
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+
+# Installing Azure CLI & Azure Arc Extensions
+echo "Installing Azure CLI"
```

Comment:
Lets please change this to the "short" version of installing Az CLI. This will be more in line with other scenarios in the Jumpstart.

```
curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
```

Reply:
Updated the commands